data-and-functions-1/src/functions/getActiveUsers.js

```diff
@@ -0,0 +1,16 @@
+const getActiveUsers = (data, users) => {
+  if (data == null || data.users == null) {
+    return null
+  } else {
+    const activeUsers = []
+    for (let i = 0; i < data.users.length; i++) {
+      const activeUser = data.users[i]
```

Comment:
Instead of calling your variable `activeUser`, call it `currentUser`. We haven't determined if it's actually an active user yet.